src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,15 @@
+package duke.command;
+
+import duke.util.TaskList;
+import duke.util.Ui;
+import duke.util.Storage;
+
+import java.util.ArrayList;
+
+/**
+ * Command class is an abstract class that different command classes inherit from.
+ */
+public abstract class Command {
```

Comment:
If you don't have any constructors or any concrete methods under `Command`, perhaps consider changing it to an interface?

Reply:
Okay, thanks for the suggestion!